spec/rubocop/cop/style/method_call_with_args_parentheses_spec.rb

```diff
@@ -444,6 +444,21 @@ def method_missing(name, ...)
           foo arg
         RUBY
       end
+
+      it 'registers an offense when using assignment with parentheses call expr follows' do
```

Comment:
I think the wording here is a bit confusing. Perhaps `when` is redundant here?

Reply:
I've fixed it. Thanks!